web/src/utils/label.ts

```diff
@@ -11,3 +11,10 @@ export const validateK8SName = (label: string) => {
   const quotesCount = (label.match(/"/g) || []).length;
   return (quotesCount == 0 || quotesCount == 2) && k8sName.test(label);
 };
+
+//TODO: remove kindToAbbr after integration of console ResourceIcon in topology library
+const abbrBlacklist = ['ASS'];
```

Comment:
:-)

Reply:
All credits to https://issues.redhat.com/browse/ODC-3630